docs/docs/policies.mdx

```diff
@@ -85,46 +86,63 @@ The Transformer Embedding Dialogue (TED) Policy is described in
 This policy has a pre-defined architecture, which comprises the
 following steps:
 
-* concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
-  step into an input vector to pre-transformer embedding layer;
+1. Concatenate user input (user intent and entities), previous system actions, slots and active forms for each time
+  step into an input vector to pre-transformer embedding layer.
 
-* feed it to transformer;
+2. Feed the input vector into a transformer.
 
-* apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step;
+3. Apply a dense layer to the output of the transformer to get embeddings of a dialogue for each time step.
 
-* apply a dense layer to create embeddings for system actions for each time step;
+4. Apply a dense layer to create embeddings for system actions for each time step.
 
-* calculate the similarity between the dialogue embedding and embedded system actions.
-  This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
+5. Calculate the similarity between the dialogue embedding and embedded system actions.
+   This step is based on the [StarSpace](https://arxiv.org/abs/1709.03856) idea.
 
-It is recommended to use `state_featurizer=LabelTokenizerSingleStateFeaturizer(...)`
-(see [Featurizers](./policies.mdx#featurizers) for details).
+It is recommended to use `SingleStateFeaturizer` (see [Featurization](#featurizers) for details).
```

Comment:
It looks like we can remove this bit, see [this Slack thread](https://rasa-hq.slack.com/archives/C016FLYV4QN/p1601896968087000) and comment about the `max_history` param. But I don't think I can do this myself as I don't have a clear understanding of the features (pun!) we're talking about 🤷